earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -126,6 +136,12 @@ void PacketGenAssetVersionImplD::UpdateChildren(
                               config.endMinifyLevel);
     }
 
+    curTime = std::chrono::high_resolution_clock::now();
+    timeDiff = std::chrono::duration_cast<std::chrono::duration<double>>(curTime - prevTime);
+    notify(NFY_PROGRESS, "UpdateChildren finished finding insets for level %d. %f seconds elapsed",
+           level, timeDiff.count());
```

Comment:
The value is recorded as seconds but is accurate to a much lower resolution.  I went with reporting seconds to avoid potential issues with wrap around in smaller units like nanoseconds

Reply:
Fair enough. Minor: for clarity sake, might want to do a duration_cast to seconds anyway

Reply:
I added a note to one of the first casts to specify the output was in seconds.  Changing to duration<double, std::ratio<1>> didn't seem to make things more clear.  I was afraid duration_cast to std::chrono::seconds would lose resolution.

Reply:
Fair enough, the comment seems sufficient here then